liminal/runners/airflow/operators/operator_with_variable_resolving.py

```diff
@@ -0,0 +1,157 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import inspect
+import logging
+import re
+
+from airflow.models import BaseOperator
+from jinja2 import Environment
+
+from liminal.runners.airflow.config import standalone_variable_backend
+
+_VAR_REGEX = '(.*){{([^}]*)}}(.*)'
+
+_BASE_OPERATOR_ATTRIBUTES = list(inspect.signature(BaseOperator.__init__).parameters.keys())
+
+
+class OperatorWithVariableResolving(BaseOperator):
```

Comment:
I think we will have issues with this kind of wrapping. There are some abstract methods like `on_kill` or `post_execute` that will not be called as we expected. for example, KubernetesPodOperator has `on_kill` method that will never be called because the original operator is wrapped, and therefore we might have some unexpected behavior.

Reply:
Right, I'll implement those